dev-packages/application-package/src/environment.ts

```diff
@@ -29,12 +29,12 @@ class ElectronEnv {
     /**
      * `true` if running in electron. Otherwise, `false`.
      *
-     * Can be called from both the `main` and the render process. Also works for forked cluster workers.
+     * Can be called from both the `main` and render processes. Also works for forked cluster workers and VS Code extension processes.
      */
     is(): boolean {
         // When forking a new process from the cluster, we can rely neither on `process.versions` nor `process.argv`.
-        // Se we look into the `process.env` as well. `is-electron` does not do it for us.
-        return isElectron() || typeof process !== 'undefined' && typeof process.env === 'object' && !!process.env.THEIA_ELECTRON_VERSION;
+        // So we look into the `process.env` as well. `is-electron` does not do it for us.
+        return isElectron() || typeof process !== 'undefined' && typeof process.env === 'object' && (!!process.env.THEIA_ELECTRON_VERSION || !!process.env.ELECTRON_RUN_AS_NODE);
```

Comment:
how this change related to a PR?

Reply:
It was used for the [previous fix](https://github.com/eclipse-theia/theia/commit/dc804c076d67a5d6285a1ff2b6166d9f6169cb0f) where `toPluginUrl` would return a different value depending on whether it was running in Electron or not. `ELECTRON_RUN_AS_NODE` covers the VS Code extension process.

So it's not necessary anymore, but it might still be an improvement (or break other things, or both, or neither). I'd keep the typo fix at least though ;)